ArduPlane/mode_takeoff.cpp

```diff
@@ -127,27 +127,26 @@ void ModeTakeoff::update()
     // reset the loiter waypoint target to be correct bearing and dist
     // from starting location in case original yaw used to set it was off due to EKF
     // reset or compass interference from max throttle
+    const float altitude = plane.current_loc.alt - start_loc.alt;
```

Comment:
Strictly you should use `get_alt_distance` on the location object which will do alt frames correctly and return in meters.

Reply:
existing code....but will change if someone gives me the correct code line